webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -0,0 +1,171 @@
+import PropTypes from 'prop-types';
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Grid,
+  Tab,
+  Tabs,
+  GridItem,
+  Badge,
+  Title,
+  Breadcrumb,
+  BreadcrumbItem,
+  Text,
+  TextVariants,
+  PageSection,
+  PageSectionVariants,
+} from '@patternfly/react-core';
+
+import Skeleton from 'react-loading-skeleton';
+import RelativeDateTime from '../../components/common/dates/RelativeDateTime';
+import StatusAlert from './Status';
+
+import { foremanUrl } from '../../../foreman_tools';
+import { get } from '../../redux/API';
+import { selectAPIResponse } from '../../redux/API/APISelectors';
+
+import Properties from './Properties';
+import ParametersCard from './Parameters';
+import InterfacesCard from './Interfaces';
+import AuditCard from './Audits';
+import ActionsBar from './ActionsBar';
+
+import './HostDetails.scss';
+
+const HostDetails = ({ match }) => {
+  const dispatch = useDispatch();
+  const [activeTab, setActiveTab] = useState(0);
+  const response = useSelector(state =>
+    selectAPIResponse(state, 'HOST_DETAILS')
+  );
+  useEffect(() => {
+    dispatch(
+      get({
+        key: 'HOST_DETAILS',
+        url: foremanUrl(`/api/hosts/${match.params.id}`),
+      })
+    );
+  }, [match.params.id, dispatch]);
+
+  useEffect(() => {
+    document.body.classList.add('transprenet-border');
```

Comment:
This causes navigation to other react pages from the host detail page to remain with this styling. why not set the class on the main component for this page instead?
Also, there's a typo - it should be `transparent-border`, but wouldn't a more descriptive name be `grey-background`? is there no pf4 class for setting this background?